style.css

```diff
@@ -0,0 +1,289 @@
+}
+
+.level__choice {
+  font: normal bold 57px/ 67px 'Roboto', sans-serif;
+  color: #fff;
+  padding: 10% 0 .6% 2.4%;
+}
+
+.level__difficulty {
+  font: normal bold 19px/ 21px 'Roboto', sans-serif;
+  color: #fff;
+  padding: 0 0 0 2.5%;
+}
+
+.game {
+  display: flex;
+  flex-direction: column;
+}
+
+.game-level {
+  position: relative;
+  outline: none;
+  background: none;
+  color: #fff;
+  font: normal bold 50px/ 59px 'Roboto', sans-serif;
+  align-self: flex-start;
+  margin-left: 6%;
+  cursor: pointer;
+  margin-bottom: 1.8%;
+}
+
+.game-level:hover {
+  -webkit-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  transform: scale(1.1);
+}
+
+#hard {
+  margin-bottom: 0;
+}
+
+/* квадратик */
+
+.active-level::before {
+  background: url(img/square.png) no-repeat;
+  background-size: cover;
+  content: "";
+  width: 25px;
+  height: 25px;
+  position: absolute;
+  top: 27%;
+  right: 110.4%;
+}
+
+.begin {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-gap: .5%;
+}
+
+.begin__button {
+  background: #d62b2e;
+  border-radius: 7px;
+  width: 54.8%;
+  height: 27.8%;
+  color: #fff;
+  font: normal bold 32px/ 37px 'Roboto', sans-serif;
+  margin: 17.6% 0 0 4.5%;
+  outline: none;
+  cursor: pointer;
+}
+
+.begin__button:hover {
+  background: #e79fa0;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 20% 20% 20%;
+}
+
+.card img {
+  border-radius: 4%;
+}
+
+.card__foto {
+  grid-column: 2/3;
+  transform: translate(31px, 19px) rotate(12deg);
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, .7);
+}
+
+.card__pic {
+  z-index: 1;
+  transform: translate(82px, 37px) rotate(39deg);
+  box-shadow: -2px 0 1px rgba(0, 0, 0, .3), 0 4px 4px rgba(0, 0, 0, .25);
+}
+
+.hidden {
+  display: none;
+}
+
+.flip {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-around;
+  width: 56.5%;
+  min-height: 1024px;
+}
+
+.card-back__wrap {
+  background: url(img/card.png);
+  align-self: center;
+  width: 218px;
+  height: 341px;
+  cursor: pointer;
+  border-radius: 4%;
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.7);
+  transition: 0.5s;
+}
+
+/* переворот */
+
+.flip-card {
+  background-color: transparent;
+  width: 218px;
+  height: 341px;
+  perspective: 1000px;
+  cursor: pointer;
+}
+
+.flip-card-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  border-radius: 4%;
+  transition: transform 0.8s;
+  transform-style: preserve-3d;
+}
+
+.flip-card-inner:hover {
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, 0.5);
+  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
+}
+
+.flip-card.active .flip-card-inner {
+ transform: rotateY(180deg);
+}
+
+.card-back,
+.card-front {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden; /* Safari */
+  backface-visibility: hidden;
+}
+
+.card-front {
+  background: url(img/gameOver.png);
+  transform: rotateY(180deg);
+}
+
+.card-front_bug {
+  background: url(img/bag.png);
+  transform: rotateY(180deg);
+} 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5876445)